<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory_Dialog</class>
 <widget class="QDialog" name="Inventory_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="ScanItem_Table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1081</width>
     <height>421</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>152</number>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Barcode</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Retail Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sale Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="AddnewInventory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Scan Barcode</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchInventory">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search Item</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Search By ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Search By Name</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
